nameinternal.py

```diff
@@ -94,6 +95,9 @@ def get_relic_stats(name: str) -> list[str]:
 def get_run_mod(name: str) -> str:
     return f'{_cache["run_mods"][name]["NAME"]} - {_cache["run_mods"][name]["DESCRIPTION"]}'
 
+def get_score_bonus(name: str) -> dict[str, str]:
```

Comment:
Technically, it's `dict[str, str | int]`, since the "bonus" key has an int as value

Reply:
Oh! Yes, please change this so it uses the new data in full_data.json -- I'm planning to deprecate and remove the JSON files under `eng/`